guides/common/modules/proc_adding-an-scc-account.adoc

```diff
@@ -2,29 +2,35 @@
 = Adding an SCC Account to {Project}
 
 .Prerequisite
-* xref:Installing_the_SCC_Manager_Plugin_{context}[SCC Manager plug-in installed] on your {Project}
+* You have installed the SCC Manager plug-in on your {Project}.
+For more information, see xref:Installing_the_SCC_Manager_Plugin_{context}[].
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Content > Content Credentials* and click *Create Content Credential*.
```

Comment:
This step should be marked as optional in my opinion.

Reply:
Applied.